reference/fleet/secure-connections.md

```diff
@@ -271,3 +273,35 @@ To encrypt traffic between {{agent}}s, {{fleet-server}}, and {{es}}:
         Don’t have an enrollment token? On the **Agents** tab in {{fleet}}, click **Add agent**. Under **Enroll and start the Elastic Agent**, follow the in-product installation steps, making sure that you add the `--certificate-authorities` option before you run the command.
 
 
+## Configure SSL/TLS using {{kib}} [fleet-server-ssl-ui-settings]
+```{applies_to}
+  stack: ga 9.1
+```
+
+You can configure SSL/TLS settings for {{fleet-server}} hosts directly in the {{fleet}} UI, without relying on CLI flags or policy overrides.
+
+To access these settings:
+
+1. In **Kibana**, go to **Management** > **Fleet** > **Settings**.
```

Comment:
We use `>` here instead of the arrow symbol. For consistency, I'd go with just one.